src/app/rpmostree-compose-builtin-tree.cxx

```diff
@@ -1469,11 +1472,6 @@ rpmostree_compose_builtin_extensions (int argc, char **argv, RpmOstreeCommandInv
       rpmostree_usage_error (context, "TREEFILE and EXTYAML must be specified", error);
       return FALSE;
     }
-  if (!opt_repo)
```

Comment:
Instead of dropping this, should we enhance it to verify that one and only one of `--repo` and `--rootfs` is provided?

Reply:
Sure, done in https://github.com/coreos/rpm-ostree/pull/3548